src/article/shared/InputActionComponent.js

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'substance'
+
+export default class InputActionComponent extends Component {
```

Comment:
Please add a description here what this Component is made for, as it is not obvious.

Reply:
Also, the name would indicate that this is an 'InputAction', which IMO is not what it is. Let's find a better name.